### PR TITLE
Update ccleaner from 1.18.30 to 2.4.141

### PR DESCRIPTION
**Pull request by LanikSJ**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/ccleaner.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "ccleaner" do
-  version "1.18.30"
-  sha256 "52295172b846c477f1c5396c9f568d602b89fad52d2fcaba394a14296e096ce6"
+  version "2.04"
+  sha256 "3369595d8e8fce701a22a18df88632556ec81ffae98872c37e100754afe1fdcb"
 
   url "https://download.ccleaner.com/mac/CCMacSetup#{version.major_minor.no_dots}.dmg"
```
Comment (LanikSJ) [inline Casks/ccleaner.rb:5]: The new version is 2.4 but I wasn't sure what variable to use here.

File: Casks/ccleaner.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cask "ccleaner" do
-  version "1.18.30"
-  sha256 "52295172b846c477f1c5396c9f568d602b89fad52d2fcaba394a14296e096ce6"
+  version "2.04"
```
Comment (SMillerDev) [inline Casks/ccleaner.rb:2]: Livecheck says
```suggestion
  version "2.4.141"
```

File: Casks/ccleaner.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 cask "ccleaner" do
-  version "1.18.30"
-  sha256 "52295172b846c477f1c5396c9f568d602b89fad52d2fcaba394a14296e096ce6"
+  version "2.4.141"
+  sha256 "3369595d8e8fce701a22a18df88632556ec81ffae98872c37e100754afe1fdcb"
 
   url "https://download.ccleaner.com/mac/CCMacSetup#{version.major_minor.no_dots}.dmg"
```
Comment (neutric) [inline Casks/ccleaner.rb:5]:
```suggestion
  url "https://download.ccleaner.com/mac/CCMacSetup#{version.major}#{version.minor.rjust(2, "0")}.dmg"
```
The minor part of the `version` seems to be represented in the `url` as a two-digit number, which requires left-padding with zero here. Done like this in `bartender` as well.

Comment (LanikSJ) [reply Casks/ccleaner.rb]: Oh nice thanks for the suggestion. 👍

Comment (neutric) [reply Casks/ccleaner.rb]: Hope it works, didn't try locally.

**Comment by neutric** [comment]
```
Error: It seems the App source '/usr/local/Caskroom/ccleaner/2.4.141/CCleaner.app' is not there.
```
This is now `Install CCleaner.pkg`, relevant documentation on how to work with pkgs is [here](https://docs.brew.sh/Cask-Cookbook#stanza-pkg).

**Pull request merged**